flash/core/model.py

```diff
@@ -110,7 +111,8 @@ def step(self, batch: Any, batch_idx: int) -> Any:
         """
         The training/validation/test step. Override for custom behavior.
         """
-        x, y = batch
+        x, y = batch['input'], batch['target']
```

Comment:
enums here ? I see that you use it in some places

Reply:
Let's move this in the children classes.

```
def training_step
       self.step((batch['input'], batch['target']))
      ....
```